testdata/cheats/Cheats.sol

```diff
@@ -64,4 +64,6 @@ interface Cheats {
     function setNonce(address,uint64) external;
     // Get nonce for an account
     function getNonce(address) external returns(uint64);
+    // Set block.chainid (newChainId)
+    function travel(uint256) external;
```

Comment:
Thoughts on calling this either `id` or `chainId` instead of `travel`? They feel much more intuitive, similar to how the cheatcode for setting the basefee is `fee`

Reply:
Renamed to `chainId`